spec/Section 7 -- Response.md

```diff
@@ -8,11 +8,11 @@ request.
 A response may contain both a partial response as well as encountered errors in
 the case that a field error occurred on a field which was replaced with {null}.
 
+A response to a GraphQL operation must be a map or an event stream of maps. The
+value of this map is described in the "Response Format" section.
 
 ## Response Format
 
-A response to a GraphQL operation must be a map.
```

Comment:
So, does this mean that in consecutive results `data` can be something else than a map? If so should we make this more clear, that in the case of stream data actually can also be a single leaf value?

Reply:
You can have a look [here](https://github.com/lukeed/dset#merging) to see how a merging strategy.